### ISSUE TITLE
Latest build process expects $GOPATH/bin in path

### ISSUE THREAD

**Issue by clintonskitson** [emccode/rexray#484]
It looks like the latest build process expects that the $PATH includes the $GOPATH/bin. This is shown by 1) the following command in the Makefile

`312		gometalinter --install --update`

2) the errors that come from the gometalinter install if the path isn't properly set

```
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
```

Extraneously I had to remove some non related packages to make gometalinter actually run. It looks like it had a pretty far reaching impact on the build process.

```
/Users/clintonkitson/go/bin/gometalinter --vendor --fast --tests --cyclo-over=16 --deadline=15s --enable=gofmt --enable=goimports --enable=misspell --enable=lll --disable=gotype --severity=gofmt:error --severity=goimports:error --errors ./core/... ./daemon/... ./rexray/... ./util/... .
rexray/cli/lstat /Users/clintonkitson/go/src/github.com/emccode/dvdcli/vendor/github.com/docker/docker/bundles/1.11.0-dev/dynbinary/docker-1.11.0-dev: no such file or directorylstat /Users/clintonkitson/go/src/github.com/emccode/dvdcli/vendor/github.com/docker/docker/bundles/1.11.0-dev/dynbinary/docker-1.11.0-dev: no such file or directory:1::error: file is not goimported (goimports)
```

**Comment by akutz** [comment]
I'll update the GOPATH bits, but you really should include it. Many projects and tools expect it in the path, but I'll def make those calls absolute.

Re: unrelated packages - Your src tree must be farked up, or something else, because those paths with dvdcli don't occur for me or on Travis. I don't know why you ran into those issues, but I didn't encounter a single one like that. I'd be interested in why they happened, but more likely it has something to do with vendoring or symlinks, or something specific to your system. I do have those projects cloned as well and they didn't interfere. Just odd...


Thanks for the

**Comment by akutz** [comment]
Oops. Thanks for the note about the GOPATH. As I said, I'll update the linter tool to be absolute.

**Comment by clintonskitson** [comment]
Problem is that even if the linter is absolute, it looks like it expects the path set properly to do its magic.

**Comment by akutz** [comment]
Then I'll update docs to make that a requirement. It should be set anyway. I'll also try to do it at top of Makefile and reexport the updated path.

**Comment by akutz** [comment]
Closed with #492
